scripts/imgtool/main.py

```diff
@@ -340,6 +345,10 @@ def convert(self, value, param, ctx):
               type=click.Choice(['128', '256']),
               help='When encrypting the image using AES, select a 128 bit or '
                    '256 bit key len.')
+@click.option('--compression', default='lzma2',
```

Comment:
does the compression is already in production state?
if not, then you may think about changing default to disabled, so the compression can be used consciously by user